File: 程式語言 V2答案檔/C++/605.cpp
#include <iostream>
using namespace std;

void sort(int[], int);
int main()
{
    int i, input[6], ans = 0;
    for (i = 0; i < 6; i++) {
        cin >> input[i];
    }
    sort(input, 6);
    for (i = 0; i < 6; i++) {
        if (i < 3)
            ans -= input[i];
        else
            ans += input[i];
    }
    cout << ans;
    return (0);
}

void sort(int list[], int length) {
    int i, j, n;
    for (i = 1; i < length; i++) {
        for (j = i; j > 0; j--) {
            if (list[j] < list[j - 1]) {
                n = list[j];
                list[j] = list[j - 1];
                list[j - 1] = n;
            }
        }
    }
}

File: Programming Language(ver.2)/C#/605.cs
using System;
using System.Collections.Generic;

public class TQC605
{
    public static void Main(string[] args)
    {
        List<int> listNum = new List<int>();
        for (int i = 0; i < 6; i++)
        {
            string input = Console.ReadLine();
            listNum.Add(Convert.ToInt32(input));
        }
        listNum.Sort();
        int sum1 = listNum[0] + listNum[1] + listNum[2];
        int sum2 = listNum[3] + listNum[4] + listNum[5];
        Console.WriteLine(sum2 - sum1);
    }
}